BabyPatterns/FeedingVC.swift

```diff
@@ -70,6 +70,23 @@ final class FeedingVC: UIViewController {
         navigationItem.backBarButtonItem?.title = " "
         let titles = FeedingType.allValues.map { $0.rawValue }
         segmentedControl.configureSegmentedBar(titles: titles, defaultSegmentIndex: 0)
+
+        let center = NotificationCenter.default
+        notificationToken = center.addObserver(forName: K.Notifications.updateFeedingsUI,
+                                               object: nil,
+                                               queue: nil)
+        { _ in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/c49e444d61917df61aa42d9cccb11a46857e6fb9/BabyPatterns/FeedingVC.swift#L78"></span>Opening braces should be preceded by a single space and on the same line as the declaration.<br />
<code>opening_brace</code> <code>FeedingVC.swift:78</code></td>
    </tr>
  </tbody>
</table>